xjalienfs.sh

```diff
@@ -41,6 +41,12 @@ for binfile in "$INSTALLROOT"/bin/*; do
 done
 rm -fv "$INSTALLROOT"/bin/*.bak
 
+# Now that alien.py is installed, we can run its tests. They need a JAliEn
+# token though, so skip them if we have none.
+if [ -n "$JALIEN_TOKEN_CERT" ] && [ -n "$JALIEN_TOKEN_KEY" ]; then
+  PATH="$INSTALLROOT/bin:$PATH" "$SOURCEDIR/tests/run_tests"
```

Comment:
Could you please use `"$SOURCEDIR/tests/run_tests ci-tests"` ?
I'm adding now this argument for running in CI environment ..

Reply:
Done, thanks! Though the CI won't pick it up yet because it's still using v1.5.8.

Reply:
yes, i will tag a new version with only the tests when i'm done